helmfile.d/charts/calico-accountant/Chart.yaml

```
@@ -2,4 +2,4 @@ apiVersion: v1
 appVersion: "1.0"
```

Nit, I think we should start updating our `Chart.yaml` files more to match what is actually used in the chart:
```suggestion
appVersion: "0.1.6.ck8s1"
```

---

Makes sense. I didn't change the `appVersion` because I did not change the application, but `1.0` is indeed a bit off when the real version is `0.1.6`. Not sure about the suffix tho, I would associate the container image update slightly more with the Chart version.

---

yea I am fine either way, as long as it is representative to the image used when deploying the chart